release-notes/add-ons/AzureKeyVaultProvider/2.0/2.0.0.md

```diff
@@ -0,0 +1,52 @@
+# Release Notes
+
+## General Availability of Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider
+_**2.0.0 released 03 March 2021**_
+
+This library contains the implementation of `Microsoft.Data.SqlClient.SqlColumnEncryptionKeyStoreProvider` for accessing Azure Key Vault, and the provider class is named `SqlColumnEncryptionAzureKeyVaultProvider`.
+
+### Added
+- Upgraded AKV Provider to use new Azure Key Vault libraries [#930](https://github.com/dotnet/SqlClient/pull/630)
```

Comment:
```suggestion
- Upgraded AKV Provider to use new Azure Key Vault libraries [#630](https://github.com/dotnet/SqlClient/pull/630)
```

Reply:
Could you also include abbreviation removal here in suggestion - let's fix that too.